src/commands/aws-ecr-login.yml

```
@@ -0,0 +1,11 @@
+description: Login into AWS ECR
+parameters:
+  aws-ecr-account-url:
+    description: The URL for your private registry
+    type: string
+steps:
+  - run:
+      command: |
+        aws ecr get-login-password | docker login --username AWS --password-stdin ${AWS_ECR_ACCOUNT_URL}
```

There's no need to declare these environment variables just to use them in the command. You can interpolate the parameters directly:
```suggestion
        aws ecr get-login-password | docker login --username AWS --password-stdin << parameters.aws-ecr-account-url >>
```